src/test/integration/java/com/commercetools/payment/BasePaymentIT.java

```diff
@@ -69,16 +71,29 @@
     protected SphereClient sphereClient;
     protected CtpFacade ctpFacade;
 
-    public void setUp() throws Exception {
+    /**
+     * Cleanup orders, carts and payments storages.
+     */
+    public void setupBeforeAll() {
+        initTenantConfigs();
+        cleanOrdersCartsPayments(sphereClient);
+    }
+
+    @Before
```

Comment:
Why do you have also in the base class? Isn't it enough to have it in the inherited classes?

Reply:
b73f595